### PR TITLE
About dojos

### PR DESCRIPTION
**Pull request by bdfinst**
Replace the splash page with the overview to reduce clicks
Describe the consortium

### CODE REVIEW

**Comment by alexherweyer** [comment]
I'm a github newb, so if i'm doing this wrong, let me know. Wording change for the second paragraph:

DevOps is often used to mean different things and people new to DevOps often focus on tooling. You cannot buy DevOps and tools represent a small portion of the solution. In our context, this is guiding principle for "DevOps"

File: overview/index.md [hunk 1]
```diff
@@ -3,11 +3,11 @@ layout: project
 title: About the DevOps Dojo Consortium
 ---
 
-The DevOps Dojo Consortium is collaboration of enterprise Dojos all seeking to improve both the outcomes of their companies and the
+We are collaboration a of enterprise Dojos all seeking to improve both the outcomes of their companies and the
```
Comment (alexherweyer) [inline overview/index.md:6]: placement of 'a' is wrong at beginning of sentence

Comment (bdfinst) [reply overview/index.md:6]: +1

**Review by alexherweyer — APPROVED** [review]
Looks good to me

**Review by drewdeal — APPROVED** [review]
This can print. Great for next iteration. :-)

**Pull request merged**